data/json/items/generic.json

```diff
@@ -2473,7 +2464,7 @@
         "price": 2000,
         "material": "veggy",
         "weight": 420,
-        "volume": 4
+        "volume":4 
```

Comment:
You should change it back to the way it was actually. Your missing a space. Should look like "volume": 4